lib/cli_common/cli_common/phabricator.py

```diff
@@ -324,3 +337,65 @@ def request(self, path, **payload):
         # Outputs result
         assert 'result' in data
         return data['result']
+
+    def load_patches_stack(self, repo, diff, default_revision='central'):
```

Comment:
You just moved this from static analysis, right? (so I don't have to review it)

Reply:
Yeap :+1: